roles/kubernetes-apps/metallb/tasks/main.yml

```diff
@@ -45,7 +39,7 @@
     src: "{{ item }}.j2"
     dest: "{{ kube_config_dir }}/{{ item }}"
     mode: 0644
-  with_items: ["metallb.yml", "metallb-config.yml", "pools.yaml", "layer2.yaml", "layer3.yaml"]
+  with_items: ["metallb.yml", "pools.yaml", "layer2.yaml", "layer3.yaml"]
```

Comment:
A nice catch for non-existent metallb-config.yml.j2 under ./roles/kubernetes-apps/metallb/templates/